src/cmd.rs

```diff
@@ -21,14 +21,14 @@ impl From<CmdError> for crate::error::Error {
         use crate::SpirvResult;
 
         match ce {
-            CmdError::BinaryNotFound(e) => Self {
+            CmdError::BinaryNotFound(err) => Self {
                 inner: SpirvResult::Unsupported,
-                diagnostic: Some(format!("failed spawn executable: {}", e).into()),
+                diagnostic: Some(format!("failed spawn executable: {err}").into()),
```

Comment:
```suggestion
                diagnostic: Some(format!("failed to spawn executable: {err}").into()),
```
(just guessing, not sure what the intent was here)